examples/slu/slurp/scripts/build_asr_tokenizer.sh

```diff
@@ -0,0 +1,12 @@
+DATA_ROOT="./slurp_data"
```

Comment:
Revert. No recipes in NeMo for the moment until we have a good way to manage them. At maximum, put it in the docstring of the train script.